src/mongocxx/test/spec/mongohouse.cpp

```diff
@@ -0,0 +1,189 @@
+// Copyright 2020 MongoDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <bsoncxx/json.hpp>
+#include <bsoncxx/stdx/string_view.hpp>
+#include <bsoncxx/string/to_string.hpp>
+#include <bsoncxx/test_util/catch.hh>
+#include <bsoncxx/types/bson_value/value.hpp>
+#include <bsoncxx/types/bson_value/view.hpp>
+#include <mongocxx/exception/operation_exception.hpp>
+#include <mongocxx/instance.hpp>
+#include <mongocxx/test/spec/monitoring.hh>
+#include <mongocxx/test/spec/operation.hh>
+#include <mongocxx/test/spec/util.hh>
+
+namespace {
+
+using namespace bsoncxx::stdx;
+using namespace bsoncxx::string;
+using namespace mongocxx::spec;
+
+using bsoncxx::builder::basic::kvp;
+using bsoncxx::builder::basic::make_document;
+
+const std::string kMongohouseURI =
+    "mongodb://mhuser:pencil@localhost/?serverSelectionTryOnce=false";
```

Comment:
Mind noting with a comment why `serverSelectionTryOnce=false` is set? The racy behavior of spawning mongohouse might not be obvious later.

Reply:
Yes, definitely!